docs/rest_api.md

```diff
@@ -533,7 +533,27 @@ Response body:
 		   "networks": {"vlan/native": "pxe", "vlan/235": "storage"}},
                   {"label": "nic2", 
 		   "macaddr": "12:34:56:78:90", 
-		   "networks":{"vlan/native": "public"}}],
+		   "networks":{}}],
+	 "metadata":{"EK":"pk"}
+	}
+
+
+Response body when run by an admin:
+
+There are two additional fields, `port` and `switch`.
+
+    {"name": "node1",
+	 "project": "project1",
+         "nics": [{"label": "nic1",
+           "port": "gi1/0/1",
+           "switch": "dell-01",
+	 	   "macaddr": "01:23:45:67:89", 
+		   "networks": {"vlan/native": "pxe", "vlan/235": "storage"}},
+                  {"label": "nic2",
+                   "port": None,
+                   "switch": None,
```

Comment:
So, this is python notation, it should be JSON. Most of it is similar, but `None` needs to be changed to `null`. Running it through `python -m json.tool` will also serve to validate it.